.circleci/config.yml

```diff
@@ -23,12 +23,15 @@ jobs:
               deno cache --reload --lock=deno.lock cli.ts 
               deno test --coverage=coverage
               deno coverage coverage --lcov --output=coverage/lcov.info
+              deno test --junit-path=./report.xml
```

Comment:
I don't think we want to run the test command twice. Could you combine both of these.
`deno test --coverage=coverage` and `deno test --junit-path=./report.xml` into a single `deno test command`, this should happen before `deno coverage`